src/OpenTelemetry.Contrib.Extensions.AWSXRay/OpenTelemetry.Contrib.Extensions.AWSXRay.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OpenTelemetry" Version="1.0.1" />
+    <PackageReference Include="OpenTelemetry" Version="1.1.0-beta1" />
```

Comment:
@srprash Please confirm if you are okay to move to next version.

Reply:
@srprash Hi. Ping to see if you want this reverted.

Reply:
Hi Cijo. Really sorry that I missed this for such a long time. 
Yes, I'm okay to update the OTel version. Will be planning the release later though.